web/bfex/components/data_pipeline/tasks/scrape.py

```diff
@@ -39,18 +40,24 @@ def run(self, data):
 
             scraper = ScraperFactory.create_scraper(faculty_directory_url, ScraperType.PROFILE)
             scrapp = scraper.get_scrapps()[0]
-            tuple = (faculty_name, scrapp)
+
+            tuple = (faculty_name,scrapp)
             tuplelist.append(tuple)
             print(tuple)
+        
 
+        #keygen = RakeApproach()
+        
+        #print(keygen.generate_keywords(scrapp.meta_data["text"]))
+        
         return tuplelist
 
 
 if __name__ == "__main__":
     task = FacultyPageScrape()
 
-    if task.is_requirement_satisfied("JNelson.Amaral"):
-        task.run("JNelson.Amaral")
+    if task.is_requirement_satisfied("Stan.Boutin"):
+        task.run("Stan.Boutin")
```

Comment:
why is this here?
I forget

Reply:
Yeah, I think I have some of these leftover as well, but lets move them to tests.